tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -61,12 +65,48 @@ Test Inference With Token Authentication
     [Documentation]    Test the inference result after having deployed a model that requires Token Authentication
     [Tags]    Sanity    Tier1
     ...    ODS-1920
-    Run Keyword And Continue on Failure    Verify Model Inference    ${MODEL_NAME}    ${INFERENCE_INPUT}    ${EXPECTED_INFERENCE_OUTPUT}    token_auth=${TRUE}
+    Run Keyword And Continue On Failure    Verify Model Inference    ${MODEL_NAME}    ${INFERENCE_INPUT}    ${EXPECTED_INFERENCE_OUTPUT}    token_auth=${TRUE}
     # Testing the same endpoint without token auth, should receive login page
     Open Model Serving Home Page
     ${out}=    Get Model Inference   ${MODEL_NAME}    ${INFERENCE_INPUT}    token_auth=${FALSE}
     Should Contain    ${out}    <button type="submit" class="btn btn-lg btn-primary">Log in with OpenShift</button>
 
+Verify Openvino_IR Model Via UI
+    [Documentation]    Test the deployment of an openvino_ir model
+    [Tags]    Sanity    Tier1
+    ...    ODS-2054
+    Open Model Serving Home Page
+    Clean Up Model Serving Page
+    Clean Up DSP Page
```

Comment:
these keywords are called by `Try Opening Create Server` 2 lines later. Are still necessary here?

Reply:
Try opening create server would just click the button if the pages are clean, so it doesn't really matter

Reply:
so you don't need to call those keywords before `Try Opening Create Server` because if the pages are not cleaned, will get cleaned in the ELSE part